packages/cli/generators/model/templates/model.ts.ejs

```diff
@@ -5,7 +5,11 @@ import {model, property} from '@loopback/repository';
 import {<%= modelBaseClass %>} from '.';
 <% } -%>
 
+<% if(disabledStrictMode) { -%>
+@model({settings: {strict: false}})
+<% } else { -%>
 @model()
+<% } -%>
 export class <%= className %> extends <%= modelBaseClass %> {
```

Comment:
Please note it's not enough to disable strict mode in model setting, the TypeScript class must be changed to allow additional properties too.

Cross-posting the relevant part from https://github.com/strongloop/loopback-next/issues/1530#issuecomment-434203916

```ts
  // Add an indexer property to allow additional data
  [prop: string]: any;
```